### PR TITLE
Updated Caption for 06-SyedAanif

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @Deepika-tmna
- [ ] @FranLara-git
- [ ] @JasonCubic
- [ ] @SyedAanif
- [ ] @daniel-eberwein
- [ ] @dilaraacinar
- [ ] @eayguen
- [ ] @gusshawstewart
- [ ] @ralsinn
- [ ] @rofofo86
- [ ] @roshvin
- [ ] @shankarananda
- [ ] @soolch
- [ ] @tgoettel9401
- [ ] @yucao89
- [ ] @andi-klickermann

### PR DESCRIPTION
**Pull request by SyedAanif**
TL;DR This PR updates the caption in 06-SyedAanif
This PR closes #2 

## Done
- create branch
- commit
- pushed to github

## TO DO

- [ ] Work on review comments 

cc @gusshawstewart

### CODE REVIEW

File: _slides/06-SyedAanif.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Software developer in a continuous learning phase!!!
```
Comment (soolch) [inline _slides/06-SyedAanif.md:6]: 100% agree to your statement 🥇 😃

Comment (rofofo86) [inline _slides/06-SyedAanif.md:6]: Is the best approach 👍  🎖️

**Review by soolch — APPROVED** [review]
Happy Journey Software Developer 👍

**Review by rofofo86 — APPROVED** [review]
LGTM.
💯

File: _slides/06-SyedAanif.md (hunk 1 above)
Comment (SyedAanif) [reply _slides/06-SyedAanif.md:6]: Thanks :)

Comment (SyedAanif) [reply _slides/06-SyedAanif.md:6]: Yes :)

**Pull request merged**